<launch>

  <node pkg="f1tenth_purepursuit" type="pure_pursuit.py" name="pp" output="screen">
        <param name="arg1" type="str" value="raceline_final"/>
        <param name="arg2" type="str" value="car_4"/>
    </node>
    <node pkg="f1tenth_purepursuit" type="gap_finder2.py" name="gf" output="screen">
    </node>
    <node pkg="f1tenth_purepursuit" type="qualify3.py" name="q3" output="screen">
    </node>
  <node pkg="rviz" type="rviz" name="rviz" args="-d $(find f1tenth_purepursuit)/src/cartographer_ros/cartographer_ros/configuration_files/map_demo.rviz" ></node>

</launch>